PhysicsTools/PatAlgos/python/slimming/miniAOD_tools.py

```diff
@@ -425,7 +425,7 @@ def _add_deepFlavour(process):
     from Configuration.Eras.Modifier_pA_2016_cff import pA_2016
     _rerun_puppijets_task = task.copy()
     _rerun_puppijets_task.add(process.puppi, process.ak4PFJetsPuppi)
-    (_run2_miniAOD_ANY | pA_2016 | pp_on_AA_2018).toReplaceWith(task, _rerun_puppijets_task)
+    (_run2_miniAOD_ANY | pA_2016 | pp_on_AA_2018 | pp_on_PbPb_run3 ).toReplaceWith(task, _rerun_puppijets_task)
```

Comment:
in the previous PR for HI jets the goal was to not run PUPPI in HI.
the goal is to clean up this dependence where possible, not to increase it (#31647)